tox.ini

```diff
@@ -94,7 +94,7 @@ commands =
     pylint --rcfile=pylintrc --output-format=parseable --ignore-paths={toxinidir}/src/pymovements/_version.py \
     {toxinidir}/src/pymovements
     pylint --rcfile=pylintrc --output-format=parseable \
-    --disable=missing-class-docstring,missing-function-docstring,protected-access {toxinidir}/tests
+    --disable=duplicate-code,missing-class-docstring,missing-function-docstring,protected-access {toxinidir}/tests/*
```

Comment:
why?

Reply:
We are already disabling it in the otherwise more restrictive pre-commit config:

https://github.com/aeye-lab/pymovements/blob/deae2fa75b97761ee3024b223fee6f82f3db633b/.pre-commit-config.yaml#L147

I don't think that duplicate test code is much of an issue, but I can change this accordingly if you like.

Reply:
🤷 no strong opinion, but I do think we have way too much logic in our tests

Reply:
you're right, I removed the disable.

There's a single test file where I added an ignore:

https://github.com/aeye-lab/pymovements/blob/ec3f99235661e251bf32b02ec3d42c9c3408208b/tests/unit/plotting/traceplot_test.py#L34

It is for a gaze fixture that is copy pasted from `tsplot_test.py`.

I will get rid of that fixture in #1003